lite/Makefile.os2

```diff
@@ -1,10 +1,13 @@
 # Makefile for OS/2 using Watcom compiler.
 #
 # wmake -f Makefile.os2
-# - builds libxmp.dll and its import lib (libxmp.lib)
+# - builds libxmplt.dll and its import lib (libxmplt.lib)
```

Comment:
If we won't have a separate -lite package, we could just generate the lib with the same file name, I don't think someone would need both versions to be available in the same system. Ideally we could have the lite configuration in the same makefile.

Reply:
That would be most ideal and it'd reduce maintenance cost.  But it'll have to be a subject of another patchset.